psydac/api/ast/fem.py

```diff
@@ -315,6 +322,22 @@ def __init__(self, expr, terminal_expr, spaces, tag=None):
             d_trials[u] = d
         # ...
 
+
+        if is_broken:
+            if is_bilinear:
+                space_domain = spaces[0].domain
+            else:
+                space_domain = spaces.domain
+
+            if isinstance(domain, Interface):
+                i = space_domain.interior.args.index(domain.minus.domain)
```

Comment:
What is `domain.minus`? And what is the difference between `domain` and `spaces.domain`?

Reply:
space_domain is the global domain the other is where you are calculating the integral

Reply:
I see, but what is `domain.minus`?

Reply:
an interface is a result of an intersection between two domains, domain_minus and domain_plus